build-system/tasks/serve.js

```diff
@@ -105,6 +109,26 @@ async function startServer(
   logServeMode();
 }
 
+/**
+ * Builds the new server by converting typescript transforms to JS
+ */
+function buildNewServer() {
+  const transformsPath = 'build-system/server/new-server/transforms';
```

Comment:
lets hoist this path out of the local function and into the module level, so it can be easily changed (if at all) and reused in line 45

Reply:
Good idea. Done.